ComponentKit/Core/CKComponent.mm

```diff
@@ -164,12 +167,15 @@ - (void)unmount
 
 - (void)_relinquishMountedView
 {
-  UIView *view = _mountInfo->view;
-  if (view) {
-    CKAssert(view.ck_component == self, @"");
-    [_scopeHandle.controller component:self willRelinquishView:view];
-    view.ck_component = nil;
-    _mountInfo->view = nil;
+  CKAssertMainThread();
+  if (_mountInfo != nullptr) {
```

Comment:
Is it worth asserting that if `_mountInfo` is `nullptr` or is the main thread assertion adequate? I'd hate to get into a situation where we avoid the crash but somehow remain in a strange state without realizing it.

I guess the question here is what could be the unintended side effects of `_mountInfo` being `nullptr` when we assume it isn't? Prior to this change we had a crash. What would happen after this change?